pkg/controller/blockdeviceclaim/blockdeviceclaim_controller.go

```diff
@@ -94,18 +93,19 @@ func (r *ReconcileBlockDeviceClaim) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, err
 	}
 
-	reqLogger.Info("BlockDeviceClaim State is:" + string(instance.Status.Phase))
 	switch instance.Status.Phase {
 	case apis.BlockDeviceClaimStatusPending:
 		fallthrough
 	case apis.BlockDeviceClaimStatusEmpty:
+		reqLogger.Info("BlockDeviceClaim State is:" + string(instance.Status.Phase))
 		err := r.claimDeviceForBlockDeviceClaim(instance, reqLogger)
 		if err != nil {
 			reqLogger.Error(err, "BlockDeviceClaim "+instance.ObjectMeta.Name+" failed")
 			return reconcile.Result{}, err
 		}
 	case apis.BlockDeviceClaimStatusInvalidCapacity:
 		// currently for invalid capacity, the BDC will remain in that state
+		reqLogger.Info("BlockDeviceClaim State is:" + string(instance.Status.Phase))
 	case apis.BlockDeviceClaimStatusDone:
```

Comment:
This log message will be displayed continously.
Scenario: 
1. BDC is created
2. BDC is bound to a BD

Then NDM operator will continously gives this logs, till the BDC is deleted.